s3_wal_test.go

```diff
@@ -161,9 +181,9 @@ func TestAppendMultiple(t *testing.T) {
 			t.Errorf("offset mismatch: expected %d, got %d", offset, record.Offset)
 		}
 
-		if string(record.Data) != string(testData[i]) {
+		if string(record.Data) != string(data[i]) {
 			t.Errorf("data mismatch at offset %d: expected %q, got %q",
-				offset, testData[i], record.Data)
+				offset, data[i], record.Data)
 		}
 	}
 }
```

Comment:
don't modify the existing test, rather add a new one

Reply:
Updated as per comment